thetis/coupled_timeintegrator_2d.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, solver):
         print_output('Coupled time integrator: {:}'.format(self.__class__.__name__))
         if not self.options.tracer_only:
             print_output('  Shallow Water time integrator: {:}'.format(self.swe_integrator.__name__))
-        if self.options.solve_tracer:
+        if self.options.tracer != {}:
```

Comment:
I wonder if there's a more succinct way to do this check, it's repeated in a few places. Maybe in `Solver2d.create_equations` you could add a boolean `self.solve_tracers = self.options.tracer != {}` to have it easily available? (Caveat is that this bool would not exist before `create_equations` is called but it's probably fine if it's used only internally. An alternative is to add something to the options itself).